Snakemake/workflow/profile/cluster.json

```diff
@@ -2,47 +2,92 @@
  "__default__": 
  {
```

Comment:
In the cluster.json, I've added the configuration for the missing rules. However, I've also changed numbers but I'm guessing this will need to be modified anyways by each user

Reply:
ok